canvas_account_admin_tools/templates/canvas_account_admin_tools/dashboard_account.html

```diff
@@ -58,6 +58,13 @@ <h4 class="list-group-item-heading">Find Course Info</h4>
           </a>
         </div>
 
+        <div class="list-group col-md-4">
+          <a type="button" class="btn btn-primary list-group-item active" href="{% url 'cross_list_courses:index' %}" class="list-group-item active">
+              <h4 class="list-group-item-heading">Cross-list Courses</h4>
+              <p class="list-group-item-text">Set up and modify cross-listings <br>&nbsp;</p>
```

Comment:
Sure. It works on different screen sizes, so no need to spend lots of time on it for this particular story, but I get a little antsy when I see `<br>`s in particular. They have their uses though! No need to over-engineer. :)